js/inputbar.js

```diff
@@ -276,10 +277,21 @@ weechat.directive('inputBar', function() {
                         // Process image
                         imgur.process(files[i], sendImageUrl);
                     }
-
                 }
             };
 
+            var deleteCallback = function () {
+                // Image got sucessfully deleted.
+                // Show toast with delete link
+                var toastDeleted = $compile('<div class="toast">Successfully deleted.')($scope)[0];
```

Comment:
that div isn't closed; also why are you using `$compile` when everywhere else we create elements without any angular involvement? I'd like to keep it consistent if there isn't a good reason to prefer `$compile`.

Reply:
Closed the div. The reason for the complile is to make ng-click work. If I do a normal onClick I'd have to put the handler function somewhere in a random place, didn't want to do that. What do?

Reply:
We could put the toast there permanently and then show and hide it.

Reply:
Nah sounds reasonable to use `$compile`, don't worry about it, I was just wondering.